keyboards/planck/keymaps/vxid/keymap.c

```diff
@@ -0,0 +1,105 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+enum planck_layers {
+  _QWERTY,
+  _LOWER,
+  _RAISE,
+  _ADJUST
+};
+
+enum planck_keycodes {
+  QWERTY = SAFE_RANGE,
+  BACKLIT,
+  EXT_PLV
+};
+
+#define LOWER MO(_LOWER)
+#define RAISE MO(_RAISE)
+#define SPCGUI MT(MOD_LGUI, KC_SPC)
+#define MINALT MT(MOD_LALT, KC_MINS)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[_QWERTY] = LAYOUT_planck_grid(
+    KC_ESC,  KC_Q,   KC_W,    KC_E,    KC_R,  KC_T,   KC_Y,   KC_U,  KC_I,    KC_O,    KC_P,    KC_BSPC,
+    KC_TAB,  KC_A,   KC_S,    KC_D,    KC_F,  KC_G,   KC_H,   KC_J,  KC_K,    KC_L,    KC_SCLN, KC_ENT,
+    KC_LSFT, KC_Z,   KC_X,    KC_C,    KC_V,  KC_B,   KC_N,   KC_M,  KC_COMM, KC_DOT,  KC_SLSH, KC_BSLS,
+    KC_LCTL, KC_GRV, KC_LPRN, KC_RPRN, LOWER, MINALT, SPCGUI, RAISE, KC_LBRC, KC_RBRC, KC_EQL,  KC_QUOT
+),
+
+[_LOWER] = LAYOUT_planck_grid(
+    _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   _______, SGUI(KC_1), SGUI(KC_2), SGUI(KC_3), _______,  _______, KC_DEL,
+    _______, KC_F5,   KC_F6,   KC_F7,   KC_F8,   _______, KC_LEFT,    KC_DOWN,    KC_UP,      KC_RIGHT, _______, _______,
+    _______, KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, LGUI(KC_1), LGUI(KC_2), LGUI(KC_3), _______,  _______, _______,
+    _______, _______, _______, _______, _______, _______, _______,    _______,    _______,    _______,  _______, _______
+),
+
+[_RAISE] = LAYOUT_planck_grid(
+    _______, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC,
+    _______, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+),
+
+[_ADJUST] = LAYOUT_planck_grid(
+    RESET,   DEBUG,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, QWERTY,  _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+)
+
+};
+
+uint32_t layer_state_set_user(uint32_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
```

Comment:
If you only have the QWERTY layer, you can remove this entire section. As well as the keycode, in general